packages/legend-manual-tests/src/__tests__/roundtrip-grammar/RoundtripGrammar.test.ts

```diff
@@ -96,6 +96,7 @@ const EXCLUSIONS: { [key: string]: ROUNTRIP_TEST_PHASES[] | typeof SKIP } = {
   'relational-property-mapping-local-property.pure': [
     ROUNTRIP_TEST_PHASES.PROTOCOL_ROUNDTRIP,
   ],
+  'merge-operation-mapping.pure': [ROUNTRIP_TEST_PHASES.PROTOCOL_ROUNDTRIP],
```

Comment:
why do we have to exclude out this phase now? Does this have to wait for some backend change or so?

Reply:
There is a difference in set implementation ids  in the mapping - not related to my change.

Reply:
@akphi  this is due to the `source` property in pure instance class mapping. That's why it's grouped there.